src/renderer/theme.store.ts

```diff
@@ -20,9 +20,11 @@
  */
 
 import { computed, observable, reaction, makeObservable } from "mobx";
-import { autoBind, boundMethod, Singleton } from "./utils";
+import { autoBind, iter, Singleton } from "./utils";
 import { UserStore } from "../common/user-store";
 import logger from "../main/logger";
+import darkThemeBase from "./themes/lens-dark.json";
+import lightThemeBase from "./themes/lens-light.json";
```

Comment:
Given that these are the only two themes we have, it seems better practice to just import them statically.